tests/UT.Set-GitHubRepoVulnerabilitySetting.Tests.ps1

```diff
@@ -5,7 +5,7 @@ Describe "Set-GitHubRepoVulnerabilitySetting tests" -Tags @("Unit") {
         Mock Invoke-GitHubRestMethod -ModuleName GitHubToolKit -MockWith { Return $null }
```

Comment:
Add an additional test to ensure that exclude repos are excluded

Reply:
Test added